scripts/claude-slide/generate_slide.py

```
@@ -0,0 +1,336 @@
+    # お化け屋敷で max_turns=10 では flaky に失敗する事象を確認したため。
+    options_kwargs: dict = {
+        "system_prompt": system_prompt,
+        "max_turns": 20,
+        "disallowed_tools": DISALLOWED_TOOLS,
+    }
+    if model:
+        options_kwargs["model"] = model
+
+    options = ClaudeAgentOptions(**options_kwargs)
+
+    print(f"  Calling Claude via Agent SDK (subscription)...")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)